.github/workflows/pr-windows.yaml

```diff
@@ -0,0 +1,21 @@
+name: "PR Windows"
```

Comment:
nit: change the workflow name and file name to match what the workflow is doing, like `name: "Build Windows executor"` / `build-executor-win.yaml`.